packages/forms/src/UIForm/utils/condition.js

```diff
@@ -16,18 +17,84 @@ function toNumber(a) {
 jsonLogic.add_operation('lowercase', lowercase);
 jsonLogic.add_operation('toNumber', toNumber);
 
+/**
+ * If in the path [] appears it will be populated
+ * with current key indices value.
+ */
+function replaceArrayNotationByIndexes(path, key) {
+	if (!path || !path.includes('[]')) {
+		return path;
+	}
+	return path
+		.split(/\.|\[/)
+		.map((part, index) => {
+			if (part === ']') {
+				return key[index];
```

Comment:
note: https://github.com/jwadhams/json-logic-js/blob/master/logic.js#L134
json logic work that way foo.0.bar